syntax = "proto3";

package Entry;

// 消息类型(收消息)
enum ENTRY_RECV_MESSAGETYPE {
    C2E_LOGIN_MESSAGE = 0;
}

// 消息类型(发消息)
enum ENTRY_SEND_MESSAGETYPE {
    E2C_LOGIN_R_MESSAGE = 0;
}

// 响应类型
enum ENTRY_MESSAGE_RESPONSE {
    OK = 0;
    PASSWD_AUTH_FAILED = 1;
    NO_ACCOUNT_RECORD = 2;
}

message C2E_LoginRequest {
    string account = 2;
    string passwd = 3;
}

message E2C_LoginResponse {
    ENTRY_MESSAGE_RESPONSE responseType = 2;
}